changelog/9195.bugfix.md

```diff
@@ -0,0 +1 @@
+DynamoDb Tracker Store correctly retrieves all sender_ids from the DynamoDb table
```

Comment:
Update DynamoDb tracker store to correctly retrieve all `sender_ids` from a DynamoDb table.